src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -28,7 +28,7 @@ vec3 F_Schlick( const in vec3 specularColor, const in float dotLH ) {
 	// Optimized variant (presented by Epic at SIGGRAPH '13)
 	float fresnel = exp2( ( -5.55437 * dotLH - 6.98316 ) * dotLH );
 
-	return ( 1.0 - specularColor ) * fresnel + specularColor;
+	return mix( specularColor, grazingColor, fresnel );
```

Comment:
With this change, this is no longer Schlick's infamous model, so the method name would need to be changed. `F_SchlickHouston`?